src/resolve/connectionResolver.ts

```diff
@@ -270,57 +270,58 @@ const querySvs =
     }
   };
 
-const listMembers =
-  (registry: RegistryAccess) =>
-  (connection: Connection) =>
-  async (query: ListMetadataQuery): Promise<RelevantFileProperties[]> => {
-    const mdType = registry.getTypeByName(query.type);
-
-    // Workaround because metadata.list({ type: 'StandardValueSet' }) returns [].
-    // Query for a subset of known StandardValueSets after all listMetadata calls.
-    if (mdType.name === registry.getRegistry().types.standardvalueset.name) {
-      shouldQueryStandardValueSets = true;
-      return [];
-    }
-
-    // Workaround because metadata.list({ type: 'BotVersion' }) returns [].
-    if (mdType.name === 'BotVersion') {
-      try {
-        const botDefQuery = 'SELECT Id, DeveloperName FROM BotDefinition';
-        const botVersionQuery = 'SELECT BotDefinitionId, DeveloperName FROM BotVersion';
-        const botDefs = (await connection.query<{ Id: string; DeveloperName: string }>(botDefQuery)).records;
-        const botVersionDefs = (
-          await connection.query<{ BotDefinitionId: string; DeveloperName: string }>(botVersionQuery)
-        ).records;
-        return botVersionDefs
-          .map((bvd) => {
-            const botName = botDefs.find((bd) => bd.Id === bvd.BotDefinitionId)?.DeveloperName;
-            if (botName) {
-              return {
-                fullName: `${botName}.${bvd.DeveloperName}`,
-                fileName: `bots/${bvd.DeveloperName}.botVersion`,
-                type: 'BotVersion',
-              };
-            }
-          })
-          .filter((b) => !!b);
-      } catch (error) {
-        const err = SfError.wrap(error);
-        getLogger().debug(`[${mdType.name}] ${err.message}`);
-        return [];
-      }
-    }
+async function listMembers(
+  registry: RegistryAccess,
+  connection: Connection,
+  query: ListMetadataQuery
+): Promise<RelevantFileProperties[]> {
+  const mdType = registry.getTypeByName(query.type);
+
+  // Workaround because metadata.list({ type: 'StandardValueSet' }) returns [].
+  // Query for a subset of known StandardValueSets after all listMetadata calls.
+  if (mdType.name === registry.getRegistry().types.standardvalueset.name) {
+    shouldQueryStandardValueSets = true;
+    return [];
+  }
```

Comment:
kinda unrelated change and doesn't have any perf impact, just simplified this b/c currying always makes it annoying to jump into funcs when attaching a debugger and didn't seem needed.